packages/wonder-blocks-core/src/hooks/use-is-mounted.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+import * as React from "react";
+
+/**
+ * Returns a function that can be called to check if the current component is still
+ * mounted or not.
+ *
+ * NOTE: Based on https://github.com/juliencrn/usehooks-ts/blob/d5f3de88cc319c790f2a4e90ba6a8904298957a5/src/useIsMounted/useIsMounted.ts
+ *
+ * @returns {() => boolean}
+ */
+export const useIsMounted = (): (() => boolean) => {
+    const isMounted = React.useRef<boolean>(false);
+    React.useEffect(() => {
+        isMounted.current = true;
+        return () => {
+            isMounted.current = false;
+        };
+    }, []);
```

Comment:
**question:** Why not use `useOnMountEffect`?

Reply:
Good idea.  That PR hasn't landed yet.  I'll land that one first and then update this file to use.